webapp/platform/types/src/users.ts

```diff
@@ -143,3 +143,16 @@ export type GetFilteredUsersStatsOpts = {
 export type AuthChangeResponse = {
     follow_link: string;
 };
+
+export type UserReport = {
+    id: string;
+    username: string;
+    email: string;
+    create_at: number;
+    display_name: string;
+    last_login_at: number;
+	last_status_at?: number;
+	last_post_date?: number;
+	days_active?: number;
+	total_posts?: number;
```

Comment:
as per the figma we might also need the role of the user because if you see actions column, it mentions either member, admin? will it be returned from the API

Reply:
<img width="386" alt="Screenshot 2023-12-07 at 12 22 32" src="https://github.com/mattermost/mattermost/assets/17708702/7e8986b4-ca12-44e3-9777-bf20f9014cd4">
in the current app

Reply:
Doesn't look like it's in the Figma, looks like we just show a menu now. @abhijit-singh Thoughts?

Reply:
Sorry I must have missed that. Will add :)

Reply:
@devinbinnie we can add the roles type here as well?

Reply:
What do you mean by the role type?